css/global.css

```diff
@@ -469,6 +469,15 @@ Utility Classes ( flexbox, wrappers,  etc.)
 .section-header {
 	padding: var(--spacing-lg);
 	text-align: center;
+	gap: var(--spacing-lg);
+}
+
+.gradient-header {
+	background: var(--grad-blue-purple);
```

Comment:
not sure if you want to keep consistency w using **background** vs. **background-img**. let me know and we can keep things consistent!